tests/conftest.py

```diff
@@ -2606,81 +2607,86 @@ def fixture_no_sample_case_id() -> str:
     return "no_sample_case"
 
 
-@pytest.fixture(name="rnafusion_sample_id")
-def fixture_rnafusion_sample_id() -> str:
-    """Returns a rnafusion sample id."""
-    return "sample_rnafusion_case_enough_reads"
-
-
-@pytest.fixture(name="rnafusion_housekeeper_dir")
-def fixture_rnafusion_housekeeper_dir(tmpdir_factory, rnafusion_dir: Path) -> Path:
-    """Return the path to the rnafusion housekeeper bundle dir."""
-    return tmpdir_factory.mktemp("bundles")
-
-
-@pytest.fixture(name="rnafusion_fastq_file_l_1_r_1")
-def fixture_rnafusion_fastq_file_l_1_r_1(rnafusion_housekeeper_dir: Path) -> str:
-    fastq_filename = Path(
-        rnafusion_housekeeper_dir, "XXXXXXXXX_000000_S000_L001_R1_001.fastq.gz"
-    ).as_posix()
+@pytest.fixture(name="fastq_file_forward")
+def fixture_fastq_file_forward(housekeeper_dir: Path) -> Path:
+    fastq_filename = Path(housekeeper_dir, "XXXXXXXXX_000000_S000_L001_R1_001.fastq.gz")
     with gzip.open(fastq_filename, "wb") as wh:
         wh.write(b"@A00689:73:XXXXXXXXX:1:1101:4806:1047 1:N:0:TCCTGGAACA+ACAACCAGTA")
     return fastq_filename
 
 
-@pytest.fixture(name="rnafusion_fastq_file_l_1_r_2")
-def fixture_rnafusion_fastq_file_l_1_r_2(rnafusion_housekeeper_dir: Path) -> str:
-    fastq_filename = Path(
-        rnafusion_housekeeper_dir, "XXXXXXXXX_000000_S000_L001_R2_001.fastq.gz"
-    ).as_posix()
+@pytest.fixture(name="fastq_file_reverse")
+def fixture_fastq_file_reverse(housekeeper_dir: Path) -> Path:
+    fastq_filename = Path(housekeeper_dir, "XXXXXXXXX_000000_S000_L001_R2_001.fastq.gz")
```

Comment:
```suggestion
def fixture_fastq_reverse_read_path(housekeeper_dir: Path) -> Path:
    fastq_file_path = Path(housekeeper_dir, "XXXXXXXXX_000000_S000_L001_R2_001.fastq.gz")
```
Add doc str, use FileExtenssion constant